src/Eccube/ServiceProvider/EccubeServiceProvider.php

```diff
@@ -317,7 +317,7 @@ public function register(BaseApplication $app)
             $types[] = new \Eccube\Form\Type\CustomerType($app); // 削除予定
 
             if (isset($app['security']) && isset($app['eccube.repository.customer_favorite_product'])) {
-                $types[] = new \Eccube\Form\Type\AddCartType($app['config'], $app['security'], $app['eccube.repository.customer_favorite_product']);
+                $types[] = new \Eccube\Form\Type\AddCartType($app['config'], $app['security'], $app['eccube.repository.customer_favorite_product'], $app);
```

Comment:
Because i use $app in function getClassCategory1 in AddCartType so i need this code